common/src/test/java/org/jboss/fuse/mvnd/common/DaemonRegistryTest.java

```diff
@@ -26,10 +26,10 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
-public class DaemonRegistryTest {
+ class DaemonRegistryTest {
```

Comment:
Why does this matter?

BTW this line and the lines below break the indentation.